contracts/common/tokens/ExitNFT.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity ^0.5.2;
+
+import { ERC721 } from "openzeppelin-solidity/contracts/token/ERC721/ERC721.sol";
+import { Registry } from '../Registry.sol';
+
+
+contract ExitNFT is ERC721 {
+  Registry internal registry;
+
+  modifier onlyWithdrawManager() {
+    require(
+      msg.sender == registry.getWithdrawManagerAddress(),
+      "UNAUTHORIZED_WITHDRAW_MANAGER_ONLY"
+    );
+    _;
+  }
+
+  constructor(address _registry, string memory _name, string memory _symbol)
+    public
+    ERC721()
```

Comment:
no need to call explicitly, empty constructors are called by default.
https://solidity.readthedocs.io/en/v0.5.7/contracts.html?highlight=constructor#arguments-for-base-constructors